### DISCUSSION TITLE
想问一下能否通过修改vite配置里面的rollup输出格式为.js，而不是修改nginx配置的MIME来部署项目？

### DISCUSSION THREAD

**Discussion by tiaopiiii** [vbenjs/vue-vben-admin#4772] [Q&A] [answered]
有参考文档[nginx部署](https://doc.vben.pro/guide/other/faq.html#nginx-%E9%83%A8%E7%BD%B2)这块来尝试，虽然在本地部署的Nginx可以运行，但是在公司服务器上配置了却不起作用，所以想问一下这样修改部署跑起来会有什么问题吗?
![屏幕截图 2024-10-31 092415](https://github.com/user-attachments/assets/63d88e78-dae3-4bd1-bade-5157278499e1)

**Comment by M69W** [comment, marked as answer]
fix: rename the Icon component to IconifyIcon to prevent name conflicts and fix type issues ([#4704](vscode-file://vscode-app/c:/Users/Z/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))

**Reply by tiaopiiii** [reply]
噢噢看到了，刚好是提交前拉的项目谢谢😂
